servers.controllers.js

```diff
@@ -103,3 +114,14 @@ exports.deleteComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getArticlebyQuery = (req, res, next) => {
```

Comment:
Don't think you need this controller, just make use of you get topics query, with your getArticles controller.